pkg/chartutil/values.go

```diff
@@ -376,3 +379,39 @@ func istable(v interface{}) bool {
 	_, ok := v.(map[string]interface{})
 	return ok
 }
+
+// PathValue takes a yaml path with . notation and returns the value if exists
+func (v Values) PathValue(ypath string) (interface{}, error) {
+	if len(ypath) == 0 {
+		return nil, error(fmt.Errorf("yaml path string cannot be zero length"))
+	}
+	yps := strings.Split(ypath, ".")
```

Comment:
Definitely not a big deal, but you might consider something like `ypsLen := len(yps)` instead of calling len(yps) a bunch of times. DEFINITELY not a big deal, though.